### ISSUE TITLE
Your whitelist to blacklist code!

### ISSUE THREAD

**Issue by begriefed** [CodeSwine/GTA5Online-Private_Public_Lobby#20]
The IP whitelist to blacklist range code is amazing and seems to be unique.  I cannot find anything to replace it.  Is there any chance you would be willing to output the inverse of the whitelist to a file, as the allowed IP list is stored, and create a command line switch to automate that process?  Windows firewall is the devil for not allowing a whitelist to take precedence over a port block.

I have a constantly updating list of IPs that I can pass to your input file but I have no way of automating the two clicks of the button to recreate the firewall rules or to even click to create the proposed blacklist range file.  Any sort of command line tool would be incredible.

Is there any automation of your program that you would be willing to implement?  Your IP range code as a stand alone would solve so many issues.  1 IP per line translated to 1 exclusionary IP range per line would be simply incredible.

**Comment by Elec0** [comment]
If I'm understanding properly, you're asking for a command-line tool to add blacklisted IPs that takes a text file as input?

**Comment by begriefed** [comment]
You create firewall rules that are the comprehensive inverse of the whitelist file settings.json.  I was asking if you would be willing to create a file, blacklist.json, that was the actual block ranges that are created in windows firewall.  I relearned how to code a bit and wrote up a perl script to do it.  It took me many hours.  Of course, when i was finished, a friend wrote it up in python in like 3 minutes.

As for the rest, I automated the processes with task scheduler and added the Take2 IPs to the whitelist.  I no longer need to shut the rules off to allow friends to join, and the rules update automatically when any IPs change.  It is a bit of a mess, but I have a fully automated public-solo-lobby, now.
